mobile/lib/src/presentation/navigation/navdrawer_widget.dart

```diff
@@ -38,6 +36,27 @@ class NavDrawerPage extends StatelessWidget {
                   .replace(Routes.recordSleepGuidePage);
             },
           ),
+          _createDrawerItem(
+            icon: Icons.analytics,
+            text: 'History',
+            onTap: () {
+              context
+                  .bloc<NavdrawerBloc>()
+                  .add(NavdrawerUpdated(NavdrawerState.NightStats));
+              ExtendedNavigator.of(context).replace(Routes.sleepHistoryPage);
+            },
+          ),
+          // Todo: link stats page to end of recording and history
```

Comment:
in another PR I imagine?

Reply:
En fait je peux enlever le comment, j'ai un basic history live qui permet d'aller vers une nuit associé.